libs/wingc/src/lib.rs

```diff
@@ -219,11 +219,24 @@ fn add_builtin(name: &str, typ: Type, scope: &mut Scope, types: &mut Types) {
 }
 
 pub fn compile(source_path: &Path, out_dir: Option<&Path>) -> Result<CompilerOutput, Diagnostics> {
-	assert!(source_path.is_file(), "Source path must be a file");
-	assert!(
-		source_path.extension().unwrap() == "w",
-		"Source file must have .w extension",
-	);
+	if !source_path.exists() {
```

Comment:
Great! Can we add a test for that?

Reply:
https://github.com/winglang/wing/pull/1552